roles/k8s-cluster/all-nodes/tasks/main.yaml

```diff
@@ -17,51 +15,69 @@
     - gnupg2
     - software-properties-common
 
-- name: Add an apt signing key for Docker
-  tags:
-  - docker
+- name: Add an apt signing key for containerd
   become: yes
   apt_key:
     url: https://download.docker.com/linux/ubuntu/gpg
```

Comment:
This url is working for containerd?

Reply:
Yes.. The containerd is download from oficial repository of the docker as recommend of site Kubernetes.
https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd

![image](https://user-images.githubusercontent.com/83663340/125348989-50996700-e333-11eb-8a37-69243d17992c.png)